CONTRIBUTORS.yaml

```diff
@@ -2247,6 +2247,12 @@ Shorla:
     email: Sogunkelu@gmail.com
     joined: 2021-10
 
+shrutikhare-git:
+    name: Shruti Khare
+    joined: 2024-12
+    email: skshrutikhare@gmail.com
+    orcid: 0000-0001-5622-2663
+    
```

Comment:
```suggestion
    location:        # add location so we can add you to the world map
      country: SG
```

Reply:
Is that where you're joining from? Or are you based somewhere else?

Reply:
If you select 'commit suggestion', then we should be able to merge this! @shrutikhare-git